src/targets/ref/CMakeLists.txt

```diff
@@ -27,14 +27,14 @@ add_library(migraphx_ref
     lowering.cpp
     gemm.cpp
 )
+target_link_options(migraphx_ref PUBLIC "-Wl,--no-as-needed")
 set_target_properties(migraphx_ref PROPERTIES EXPORT_NAME ref)
 rocm_set_soversion(migraphx_ref ${MIGRAPHX_SO_VERSION})
 
 find_path(BLAZE_INCLUDE blaze/Blaze.h)
-find_package(Threads)
 
 rocm_clang_tidy_check(migraphx_ref)
-target_link_libraries(migraphx_ref migraphx Threads::Threads)
+target_link_libraries(migraphx_ref PUBLIC migraphx)
```

Comment:
I dont think `Threads` should be dropped.

Reply:
okay, i can't find how and where it is used for `ref`. `Threads` links to `libmigraphx.so` already.